<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>353</width>
    <height>296</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>353</width>
    <height>296</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>353</width>
    <height>296</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(151, 151, 151);
item {
background-color: transparent;
}
item:selected {
background-color: rgb(163, 3, 47);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>331</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: &quot;Segoe UI&quot;;
    font-size: 10pt;
    border: 1px solid;
    border-color: rgb(192,192,192); 
    border-radius: 3px; 

    color: black;
    font: bold;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>220</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: &quot;Segoe UI&quot;;
    font-size: 8pt;
    border: 1px solid;
    border-color: rgb(192,192,192); 
    border-radius: 3px; 
 background-color: rgb(230,230,230);
    color: black;
    font: bold;</string>
    </property>
    <property name="text">
     <string>Обрати</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>331</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">font-family: &quot;Segoe UI&quot;;
    font-size: 12pt;
    border: 1px solid;
    border-color: rgb(192,192,192); 
    border-radius: 3px; 

    color: black;
    font: bold;</string>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">font-family: &quot;Segoe UI&quot;;
    font-size: 12pt;
    border: 1px solid;
    border-color: rgb(192,192,192); 
    border-radius: 3px; 

    color: black;
    font: bold;</string>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>331</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QToolButton" name="toolButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">font-family: &quot;Segoe UI&quot;;
    font-size: 8pt;
    border: 1px solid;
    border-color: rgb(192,192,192); 
    border-radius: 3px; 
 background-color: rgb(230,230,230);
    color: black;
    font: bold;</string>
       </property>
       <property name="text">
        <string>Список</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">font-family: &quot;Segoe UI&quot;;
    font-size: 8pt;
    border: 1px solid;
    border-color: rgb(192,192,192); 
    border-radius: 3px; 
 background-color: rgb(230,230,230);
    color: black;
    font: bold;</string>
       </property>
       <property name="text">
        <string>Список</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>331</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="styleSheet">
        <string notr="true">font-family: &quot;Segoe UI&quot;;
    font-size: 9pt;
    border: 1px solid;
    border-color: rgb(192,192,192); 
    border-radius: 3px; 

    color: black;
    font: bold;</string>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="styleSheet">
        <string notr="true">font-family: &quot;Segoe UI&quot;;
    font-size: 9pt;
    border: 1px solid;
    border-color: rgb(192,192,192); 
    border-radius: 3px; 

    color: black;
    font: bold;</string>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>353</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 lightgray, stop:1 darkgray);
item {
spacing: 3px;
padding: 1px 4px;
background: transparent;
border-radius: 4px;
}
item:selected {
background: #a8a8a8;
}
item:pressed {
background: #888888;
}</string>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 lightgray, stop:1 darkgray);</string>
   </property>
  </widget>
  <action name="action">
   <property name="text">
    <string>Вийти з програми</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
